src/app/rpmostree-ex-builtin-jigdo2commit.c

```diff
@@ -24,6 +24,9 @@
 #include <glib-unix.h>
 #include <gio/gunixoutputstream.h>
 #include <libdnf/libdnf.h>
+// For the jigdo Requires parsing
+#include <libdnf/dnf-reldep-private.h>
+#include <libdnf/dnf-sack-private.h>
```

Comment:
Hmm, sounds like we need an API for reldeps. Let's open an issue upstream to track that? Eventually, we want to move to a shared lib, right?

Reply:
OK https://github.com/rpm-software-management/libdnf/issues/397